python/lsst/ts/observatory/control/mock/mtcs_mock.py

```diff
@@ -151,57 +149,51 @@ async def mount_telemetry(self):
                 == salobj.State.ENABLED
             ):
 
                 # Safely initilize data
-                if not self.controllers.mtmount.tel_Azimuth.has_data:
-                    self.controllers.mtmount.tel_Azimuth.set(
-                        Azimuth_Angle_Set=0.0, Azimuth_Angle_Actual=0.0
+                if not self.controllers.mtmount.tel_azimuth.has_data:
+                    self.controllers.mtmount.tel_azimuth.set(
+                        angleSet=0.0, angleActual=0.0
                     )
 
-                if not self.controllers.mtmount.tel_Elevation.has_data:
-                    self.controllers.mtmount.tel_Elevation.set(
-                        Elevation_Angle_Set=0.0, Elevation_Angle_Actual=0.0
+                if not self.controllers.mtmount.tel_elevation.has_data:
+                    self.controllers.mtmount.tel_elevation.set(
+                        angleSet=0.0, angleActual=0.0
                     )
 
-                if not self.controllers.newmtmount.evt_target.has_data:
-                    self.controllers.newmtmount.evt_target.set()
+                if not self.controllers.mtmount.evt_target.has_data:
+                    self.controllers.mtmount.evt_target.set()
 
                 az_induced_error = np.random.normal(0.0, 1e-7)
                 el_induced_error = np.random.normal(0.0, 1e-7)
 
-                az_set = self.controllers.mtmount.tel_Azimuth.data.Azimuth_Angle_Set
-                el_set = self.controllers.mtmount.tel_Elevation.data.Elevation_Angle_Set
+                az_set = self.controllers.mtmount.tel_azimuth.data.angleSet
+                el_set = self.controllers.mtmount.tel_elevation.data.angleSet
 
-                az_actual = (
-                    self.controllers.mtmount.tel_Azimuth.data.Azimuth_Angle_Actual
-                )
-                el_actual = (
-                    self.controllers.mtmount.tel_Elevation.data.Elevation_Angle_Actual
-                )
+                az_actual = self.controllers.mtmount.tel_azimuth.data.angleActual
+                el_actual = self.controllers.mtmount.tel_elevation.data.angleActual
 
                 az_dif = salobj.angle_diff(az_set, az_actual)
                 el_dif = salobj.angle_diff(el_set, el_actual)
                 in_position_elevation = np.abs(el_dif) < 1e-1 * u.deg
                 in_position_azimuth = np.abs(az_dif) < 1e-1 * u.deg
 
                 if self.acting:
-                    self.controllers.newmtmount.evt_axesInPosition.set_put(
+                    self.controllers.mtmount.evt_axesInPosition.set_put(
                         elevation=in_position_elevation, azimuth=in_position_azimuth
                     )
 
-                self.controllers.mtmount.tel_Azimuth.set_put(
-                    Azimuth_Angle_Actual=az_actual + az_induced_error + az_dif.deg / 1.1
+                self.controllers.mtmount.tel_azimuth.set_put(
+                    angleActual=az_actual + az_induced_error + az_dif.deg / 1.1
```

Comment:
Pre-existing issue but please add a brief code comment about the constant 1.1 (and consider a named constant for clarity).

Reply:
Ok, I added a constant and comment with clarification.